### ISSUE TITLE
Problem with config?

### ISSUE THREAD

**Issue by efedorenko** [hudochenkov/postcss-sorting#21]
Hi Aleks,

I've been strugling with adding option `preserve-empty-lines-between-children-rules` (or `empty-lines-between-children-rules`, or `empty-lines-between-media-rules` for that matter) to my config file. Here is my config and how I run it: https://github.com/wildbit/postcss-sorting-config-wildbit

As long as config JSON has only `sort-order` option it works fine, but when I add anything else it fails:

```
gulp cleanup
[11:55:58] Using gulpfile ~/Sites/beanstalk/gulpfile.js
[11:55:58] Starting 'cleanup'...

events.js:154
      throw er; // Unhandled 'error' event
      ^
TypeError: Cannot read property 'type' of undefined
    at getApplicableNode (/Users/eugene/Sites/beanstalk/node_modules/postcss-sorting/index.js:188:14)
    at getApplicableNode (/Users/eugene/Sites/beanstalk/node_modules/postcss-sorting/index.js:193:10)
    at /Users/eugene/Sites/beanstalk/node_modules/postcss-sorting/index.js:306:25
    at Rule.each (/Users/eugene/Sites/beanstalk/node_modules/postcss/lib/container.js:68:22)
    at /Users/eugene/Sites/beanstalk/node_modules/postcss-sorting/index.js:281:10
    at /Users/eugene/Sites/beanstalk/node_modules/postcss/lib/container.js:81:26
    at Root.each (/Users/eugene/Sites/beanstalk/node_modules/postcss/lib/container.js:68:22)
    at Root.walk (/Users/eugene/Sites/beanstalk/node_modules/postcss/lib/container.js:80:21)
    at /Users/eugene/Sites/beanstalk/node_modules/postcss-sorting/index.js:219:7
    at LazyResult.run (/Users/eugene/Sites/beanstalk/node_modules/postcss/lib/lazy-result.js:206:20)

npm ERR! Darwin 15.4.0
npm ERR! argv "/Users/eugene/.nvm/versions/node/v5.11.0/bin/node" "/Users/eugene/.nvm/versions/node/v5.11.0/bin/npm" "run" "cleanup"
npm ERR! node v5.11.0
npm ERR! npm  v3.8.6
npm ERR! code ELIFECYCLE
npm ERR! beanstalk-app@0.2.0 cleanup: `gulp cleanup`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the beanstalk-app@0.2.0 cleanup script 'gulp cleanup'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the beanstalk-app package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     gulp cleanup
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs beanstalk-app
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls beanstalk-app
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/eugene/Sites/beanstalk/npm-debug.log
```

I tried to look at other configs, but all of the default ones have a single `sort-order` option in them.

Would really appreciate if you can point me in a right direction!

**Comment by hudochenkov** [comment]
Have you try this?

```json
{
	"sort-order": [
		"..."
	],
	"preserve-empty-lines-between-children-rules": true
}
```

If you do and plugin fails, can you share a file what cause plugin to fail?

**Comment by efedorenko** [comment]
With your reduced config I found a piece of SCSS that breaks postcss-sorting:

```scss
  .form-subsection-tbl .tbl-name {
    /* obscure stuff */
    #page & {
      width: 166px;
    }
  }
```

It's actually the comment that makes it fail. Hope that helps :)

**Comment by hudochenkov** [comment]
Thank you! I'll fix this today.

**Comment by hudochenkov** [comment]
Fixed in `1.4.1`.

**Comment by efedorenko** [comment]
Works great! Thank you 👍
